scripts/test_solidity_lsp.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3.8
```

Comment:
I don't think this is a good way to express a dependency on a particular version of Python. It will not work if you have e.g. python 3.9 installed.
```suggestion
#!/usr/bin/env python3
```

Reply:
quite funnily, I did that even without having read your comment here :-) I agree, actually. ;-)

Reply:
LSP testing is already in CI.

With regards to using python test suite, as said, I'd complete the test cases first, in order to get that through within next 2 days. Moving to another test infrastructure is IMHO not that pressing, because the current works.

Reply:
That way you're just writing from scratch some things that the unit test framework can give you for free. For example test discovery or diffing. But, well, if you already have it then I guess rewriting in unittest might just as well be done in a separate PR. I could help with that.